mmv1/products/datafusion/go_instance.yaml

```diff
@@ -0,0 +1,276 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# resource:
+name: 'Instance'
+baseurl: 'projects/{{project}}/locations/{{region}}/instances'
+createurl: 'projects/{{project}}/locations/{{region}}/instances?instanceId={{name}}'
+updateverb: 'PATCH'
+description: 'Represents a Data Fusion instance.'
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/data-fusion/docs/'
```

Comment:
I modified this line to make the parsing work.

The original line is 
```
- 'Official Documentation': 'https://cloud.google.com/data-fusion/docs/'
```
 and it gives the error `cannot unmarshal !!seq into map[string]string`